build/media_source/templates/administrator/atum/scss/pages/_com_cpanel.scss

```
@@ -194,8 +194,7 @@
   @include color-mode(dark) {
     .com_cpanel {
       .card {
-        border: 1px solid rgba(255, 255, 255, .1);
-        box-shadow: 1px 1px 2px rgba(0, 0, 0, .1);
+        box-shadow: 0 0 0 1px rgba(255, 255, 255, .1);
```

```suggestion
        box-shadow: 0 0 0 1px rgba(255, 255, 255, .1), 1px 1px 2px rgba(0, 0, 0, .1);
```

Would prefer not to lose the box shadow originally set by the template team and set both.

---

OK for me